get-users/creative-guidelines.md

```diff
@@ -0,0 +1,40 @@
+# Creative Guidelines 
+
+
+## Ad Units
+
+Promotional creatives can be banners or full screen interstitials. To learn more about what types of ads we support, and how they are displayed, check out our [ad units](doc/make-money/ad-overview) documentation.
+
+## Performance
+
+You want to optimize your creatives for top performance. If you achieve a higher TTR (tap through rate) and CR (conversion rate) your creative will waste less valuable impressions.  This will result in better ads being displayed on your app, with you earning more tap credits. 
```

Comment:
Do you mean to say "... your creative will waste fewer valuable impressions"? The way it is written now, the less seems to read "you will waste impressions that are less valuable".